src/SubjectHead/style.js

```diff
@@ -0,0 +1,119 @@
+import { makeStyles } from '@material-ui/core/styles';
+import colors from '../styles/colors';
+
+const useStyles = makeStyles({
+    body:{ 
+        fontSize:"62.5%", 
+    },
+    container: {
+        // background:colors.blue,
+        background: '#191C4D',
+        width: '100%',
+        height: '37.56rem',
+        display:'flex',
+        flexDirection: 'row',
+
+  },
+  Head_left:{
```

Comment:
Be consistence while using classNames
Avoid using `_`, generic names and keep the first letter in lowercase like `subjectHeadLeft`